src/shared/types.ts

```diff
@@ -35,7 +35,4 @@ export interface TeamResult {
   isAssassinated: boolean;
 }
 
-export interface MeetingToken {
```

Comment:
Because we now pass the game ID to `tryGetMeetingToken()`, we do not need to have it in the token cookie itself.